packages/components/psammead-content-anchor/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.0-alpha.25 | [PR#4561](https://github.com/bbc/psammead/pull/4561) bump dependencies |
```

Comment:
Not related to this PR but any reason why this package is versioned a bit differently (has `alpha` as suffix)?

Reply:
So we follow the [semver](https://semver.org/) for our dependancy versioning. When a package is marked as `alpha` or `beta` it means the package is in a prerelease state, which shows users that the packages aren't fully ready for. production usage. Here's a doc explaining it's usage in more detail within Psammead https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md#alpha-components

Reply:
Ah cool, thanks.